### PR TITLE
fix(cat-ci): Flutter broken CI cache

### PR DESCRIPTION
**Pull request by stevenj**
# Description

Adds a reliable version number to browser artifacts required by the flutter builder.

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: earthly/flutter/installer/Earthfile [hunk 1]
```diff
@@ -1,6 +1,13 @@
 VERSION 0.8
 
-ARG --global BASE_URL = "https://github.com/input-output-hk/catalyst-storage/raw/main/web-browsers"
+# cspell: words Rxxxx
+
+# DO NOT USE `main` branch.
+# When updating the artifacts, update them in `main` and then create a sequentially numbered branch from it called `Rxxxx`
+# Then replace that Release version number in the path below.
+# This ensures that the tools are properly cached, and will also be re-installed when the upstream artifact is updated.
+# If a branch is no longer used by CI, delete it.
```
Comment (dtscalac) [inline earthly/flutter/installer/Earthfile:9]: Deleting "unused" branches might break older versions of CI that still depends on them.

If a branch is unused in the newest CI version but was used in previous CI version then after removing branch from catalyst-storage the previous CI version gets broken.

Shouldn't these branches stay alive as long as CI version that needs them stays alive? Do we remove CI versions or they stay forever?

Comment (stevenj) [reply earthly/flutter/installer/Earthfile:9]: Thats what the last comment `If a branch is no longer used by CI, delete it1 means.
That we shouldn't delete the branch until we are sure its not in use by CI anymore.
If someone is on an old branch locally, they can easily update.

Comment (stevenj) [reply earthly/flutter/installer/Earthfile:9]: I see what your saying.
In theory the CI versions stay forever, they aer just tags.
But we never use OLD Ci version for anything except "information".  And its not an external resource, so no one outside our group should be using cat-ci v1.3, and if they do, thats on them.

Comment (stevenj) [reply earthly/flutter/installer/Earthfile:9]: In theory, old CI branches may already be broken.  Even upstream container images and tool versions disappear (or are yanked).  So there is no guarantee that any old (currently unused) ci version is still able to retrieve the assetts it relies on.

Comment (dtscalac) [reply earthly/flutter/installer/Earthfile:9]: Alright, thanks for clarification

**Review by dtscalac — APPROVED** [review]
LGTM

**Pull request merged**